permission_handler/example/lib/plugin_example/permission_list.dart

```diff
@@ -17,9 +17,10 @@ class PermissionList extends StatelessWidget {
                 if (Platform.isIOS) {
                   return permission != Permission.unknown &&
                       permission != Permission.sms &&
-                      //permission != Permission.storage &&
+                      permission != Permission.storage &&
                       permission != Permission.ignoreBatteryOptimizations &&
-                      permission != Permission.accessMediaLocation;
+                      permission != Permission.accessMediaLocation &&
+                      permission != Permission.activityRecognition;
```

Comment:
on iOS it's the same as Permission.sensor so the Android variant shoudn't be visible on iOS.